# failing callback still generates useful body

    [1] "`body` callback defined in req_error() failed with error:"
    [2] "This is an error!"                                        

